receiver/signalfxreceiver/receiver.go

```diff
@@ -165,21 +171,32 @@ func (r *sfxReceiver) Shutdown(context.Context) error {
 	return err
 }
 
-func (r *sfxReceiver) readBody(ctx context.Context, resp http.ResponseWriter, req *http.Request) ([]byte, bool) {
+func (r *sfxReceiver) readBody(ctx context.Context, resp http.ResponseWriter, req *http.Request, events bool) ([]byte, bool, bool) {
```

Comment:
Can we break down this func? Returning 3 arguments is a bit hard to read

Reply:
made some changes to make it more readable